repair/row_level.cc

```diff
@@ -2486,10 +2487,13 @@ class row_level_repair {
                     send_missing_rows_to_follower_nodes(master);
                 }
             } catch (std::exception& e) {
+                std::string err(e.what());
+                if (err.find("Can't find a column family") == 0) {
```

Comment:
string matching is brittle. why not catch the no_such_column_family exception?

Reply:
or rethrow here and catch it to handle the specific exception

Reply:
I hate the string match here too, but the error is from rpc. rpc loses the exception type, it is std::runtime_error.

Reply:
got it, thanks